tests/jax/single_chip/models/example_model/only_kwargs/test_example_model_only_kwargs.py

```
@@ -31,6 +31,10 @@ def _get_input_activations(self) -> Sequence[jax.Array]:
         act = jax.numpy.ones(act_shape)
         return [act]
 
+    # @override
+    def _get_input_parameters(self) -> PyTree:
+        return ()
```

You could return `()` by default instead of raising an exception in `ModelTester._get_input_parameters`. That plus the `FlaxPreTrainedModel` default behaviour seems to cover a lot of cases. You can override it anyway if you need to in derived class.

---

Not really, `()` is legit only for the example model. We wan't to make sure user doesn't forget to implement this method, I think it is better to raise the exception in non-default case.